backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
+        first_name: {type: 'varchar(30)'},
+        last_name: {type: 'varchar(150)'},
+        password: {type: 'varchar(128)'},
+        bio: {type: 'varchar(500)'}
+    });
+
+/*****************************************************************************/
+    pgm.createTable('activities', {
+        id: 'id',
+        name: {type: 'varchar(250)', notNull: true, unique: true}
+    });
+
+    pgm.createIndex('activities', 'name');
+
+/*****************************************************************************/
+    pgm.createTable('places', {
+        id: 'id',
+        country: {type: 'varchar(250)', notNull: true},
+        city: {type: 'varchar(250)', notNull: true}
+    });
+
+    pgm.addConstraint('places', 'places_event_country_city_uniq', {
+        unique: ['country', 'city']
+    });
+
+/*****************************************************************************/
+    pgm.createTable('events', {
+        id: 'id',
+        name: {type: 'varchar(250)', notNull: true},
+        description: {type: 'text'},
+        activity: {
+            type: 'integer',
+            notNull: true,
+            references: '"activities"',
+            deferrable: true,
+            deferred: true
+        },
+        place: {
+            type: 'integer',
```

Comment:
I would allow it to be null for online events and events with location: TBD

Reply:
Yes, this is a good idea.